### PR TITLE
Migrate scenarios to DAML Script

### PR DESCRIPTION
**Pull request by cocreature**

### CODE REVIEW

File: .dlint.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+- ignore: {name: Redundant pure }
```
Comment (cocreature) [inline .dlint.yaml:1]: This triggers incorrectly in the presence of `ApplicativeDo` where you need the `return` at the end or you end up with an `Action` constraint.

File: src/main/daml/Test/DA/RefApps/KnowYourCustomer/Utils.daml [hunk 2]
```diff
@@ -41,3 +41,9 @@ createDataStream' publisher consumer (PublisherRelationship relationship pubRole
   proposal <- publisher `submit` exerciseCmd request DataStreamRequest_Propose with price
   license <- consumer `submit` exerciseCmd proposal DataLicenseProposal_Accept
   publisher `submit` exerciseCmd pubRole RegisterLicense with licenseId = license
+
+
+queryContractKey : forall t k. (TemplateKey t k, Eq k) => Party -> k -> Script (Optional (ContractId t, t))
```
Comment (cocreature) [inline src/main/daml/Test/DA/RefApps/KnowYourCustomer/Utils.daml:46]: We might move this in the standard library (in a more efficient form) but for now this does the job.

**Comment by gaborh-da** [comment]
Locally, lots of tests fail.

**Comment by cocreature** [comment]
> Locally, lots of tests fail.

Bumped the LF version, now `mvn verify` passes for me locally.

**Comment by cocreature** [comment]
Closed in favor of #20

**Pull request closed without merging**